getting-started/README.md

```diff
@@ -73,14 +73,13 @@ Create a file named `tracing.js` and add the following code:
 ```javascript
 'use strict';
 
-const opentelemetry = require("@opentelemetry/api");
 const { NodeTracerProvider } = require("@opentelemetry/node");
 
 const provider = new NodeTracerProvider({
   logLevel: opentelemetry.LogLevel.ERROR
```

Comment:
If opentelemetry isn't required, where does opentelemetry.LogLevel.ERROR come from? Also I think LogLevel is in "@opentelemetry/core", not "@opentelemetry/api".

Reply:
it is in API but got catch I will put it back

Reply:
good catch